src/types/data.ts

```diff
@@ -28,12 +28,42 @@ export type EntityType = 'nemeses' | 'mages' | 'cards'
 export type Entity = ICard | ICreature
 
 export type NemesisCardType = 'Power' | 'Minion' | 'Attack'
+export type NemesisCardTier = 1 | 2 | 3
 
+export type MinionCard = {
+  type: 'Minion'
+  hp: number
+  shields?: number
+}
+
+export type PowerCard = {
+  type: 'Power'
+  power: number
+}
+
+export type AttackCard = {
+  type: 'Attack'
+}
+
+export type BasicNemesisCard = {
+  id: string
+  name: string
+  expansion: string
+  tier: NemesisCardTier
+  effect: string
+  unleashCount?: number
```

Comment:
We will keep unleashCount, playerDamage and graveholdDamage in mind, but we will remove them for the initial implementation of the feature and add them later.